### PR TITLE
removal, reordering and rewording for a coherent, comp[ete story minus repetition

### PR DESCRIPTION
**Pull request by adinn**
I have reworked the main body of text to re-order a few things, avoid some repetitions and give the whole thing a consistent narrative. Also, I changed all occurrences of CDS and CDS cache to AOT and AOT cache since we need to avoid presenting the Leyden solution as a variation on CDS. The only references to CDS which are left refer to the historical implementation as a step towards the Leyden AOT cache.

### CODE REVIEW

File: _posts/2024-07-19-quarkus-and-leyden.adoc [hunk 1]
```diff
@@ -133,20 +106,27 @@ JiT will also lazily upgrade compiled code after it has been executed very many
...
 
 == Current status of Project Leyden
 
-There are already experimental https://jdk.java.net/leyden/[early-access builds of Leyden] that can be tested based on https://openjdk.org/jeps/8315737[this draft JEP about Ahead-of-Time Class Linking]. With the https://www.youtube.com/watch?v=lnth19Kf-x0[Leyden Project], the training run idea has been extended to a wider range of data structures, creating the Cache Data Store(CDS). Now the training data contains:
+There are already experimental https://jdk.java.net/leyden/[early-access builds of Leyden] that can be tested based on https://openjdk.org/jeps/8315737[this draft JEP about Ahead-of-Time Class Linking]. With the https://www.youtube.com/watch?v=lnth19Kf-x0[Leyden Project] *** this link is oto out of date -- maybe use the 20204 leyden talk***, the training run idea has been extended to a wider range of data structures embedded in the new AOT cache. Now the training data contains:
```
Comment (Delawen) [inline _posts/2024-07-19-quarkus-and-leyden.adoc:123]: Do we have the link available already?

Comment (adinn) [reply _posts/2024-07-19-quarkus-and-leyden.adoc:123]: Not yet. The 2024 JVMLS is next week (https://openjdk.org/projects/mlvm/jvmlangsummit/agenda.html) and the Leyden talk will be on Monday afternoon. The talk may not be uploaded straight away though.

Perhaps we might link to the EA release page instead? The EA is advertised at https://jdk.java.net/leyden/

Comment (adinn) [reply _posts/2024-07-19-quarkus-and-leyden.adoc:123]: n.b. the latest EA (and most notably the release notes hung off that link) does not use the AOTCache terminology introduced in the latest JEP draft. However a new EA is being prepared ready for the JVLMS talk which will use that terminology.

**Comment by Sanne** [comment]
Thanks! I've merged this locally, then reworked a little and squashed all (so it looks like conflicting - but it's merged).

**Pull request closed without merging**